tests/model_fields/tests.py

```diff
@@ -321,5 +322,11 @@ def test_get_choices_reverse_related_field(self):
         )
         self.assertChoicesEqual(
             field.get_choices(include_blank=False, limit_choices_to={}),
-            [self.bar1, self.bar2],
+            [self.bar1, self.bar2, self.bar3],
+        )
+
+    def test_get_choices_distinct(self):
+        self.assertChoicesEqual(
+            self.field.get_choices_distinct(include_blank=False, limit_choices_to={'bars__b__startswith': 'a'}),
```

Comment:
```suggestion
            self.field.get_choices(include_blank=False, limit_choices_to={'bars__b__startswith': 'a'}),
```

Reply:
I made the corrections... let me know if there is something else.